lib/openstax/aws/stack.rb

```diff
@@ -220,6 +222,18 @@ def apply_change_set(params: {}, wait: false)
       change_set
     end
 
+    def revert_to_previous_change_set(wait: false)
+      logger.info("**** DRY RUN ****") if dry_run
+
+      if @previous_parameters
+        logger.info("Applying previous parameters...")
+        apply_change_set(params: @previous_parameters, wait: wait)
+        @previous_parameters = nil
+      else
```

Comment:
The previous change set could also have had a different stack template.  So we'd need to use that too? (the change set we are trying to revert could be using both new parameter values and a new template). Code like https://github.com/openstax/aws-ruby/blob/master/lib/openstax/aws/stack.rb#L59-L60 is maybe helpful.

Reply:
Looks good, I've added some fixes and updated cassettes: https://github.com/openstax/aws-ruby/pull/71